tests/sentry/workflow_engine/migration_helpers/test_migrate_rule.py

```diff
@@ -155,7 +186,9 @@ def test_update_issue_alert_no_actions(self):
             condition_result=True,
         ).exists()
 
-        assert DataConditionGroupAction.objects.filter(condition_group=if_dcg).count() == 0
+        dcg_actions = DataConditionGroupAction.objects.get(condition_group=if_dcg)
+        action = dcg_actions.action
+        assert action.type == Action.Type.PLUGIN  # tested fully in test_migrate_rule_action.py
```

Comment:
why is the type plugin here if `self.issue_alert`'s action is `SlackNotifyServiceAction`?

Reply:
this is a test for updating the issue alert with a payload we extract from the request
https://github.com/getsentry/sentry/pull/84467/files#diff-3696b5050b59b40e6e9476eb63213a40a67dd3fa5b8364c9db11a13fb5c2e97bR142-R153

it's weird because we have `ProjectRuleCreator` and `ProjectRuleUpdater` classes we use to update Rule objects